friends/src/App.css

```diff
@@ -0,0 +1,108 @@
+.App {
+  text-align: center;
+}
+
+.List{ 
+  width: 40%;
+  margin: 0 auto;
+  margin-bottom: 30px;
+}
+
+.friend{
+  border: 1px solid green;
+  margin-top: 20px;
+  width: 100%;
+  display:flex;
+  flex-direction: row;
+  justify-content: left;
+  align-items: center;
+}
+.friend button{
+  height: 30px;
+}
+.friend .card{
+  border: 1px dashed lightgray;
+  padding: 0 10%;;
+  /* text-align: left; */
+  width: 470px;
+  margin-right: 50px;
+}
+.form{
+  margin: 40px 0 20px;
+}
+.App .nav{
+  
+  
+  display:flex;
```

Comment:
It looks like there's some odd spacing throughout here; try to make it consistent.